test/libsolidity/semanticTests/externalSource/_source_name_starting_with_dots/dir/contract.sol

```diff
@@ -0,0 +1,2 @@
+import "./a.sol";
+import "../b.sol";
```

Comment:
To reduce the number of files I think you could just put this directly in `_source_name_starting_with_dots.sol`.

Reply:
This is not working, because, if I understood correctly, `./` and `../` will be replaced with a value in relation to the path of the contract. That means if we put this directly in `source_name_starting_with_dots.sol`, it would not work. `import "./a.sol";` becomes `import "a.sol";`  and  `import "../b.sol";` becomes `import "b.sol";`. We didn't defined these files.

Reply:
Ah, you're right, sorry :) It has to be in `contract.sol`.